kedro/extras/extensions/ipython.py

```diff
@@ -44,6 +46,14 @@ def reload_kedro(path, env: str = None, extra_params: Dict[str, Any] = None):
     from kedro.framework.session.session import _activate_session
     from kedro.framework.startup import bootstrap_project
 
+    # If a path is provided, set it as default for subsequent calls
+    global project_path
+    if path:
+        project_path = Path(path).expanduser().resolve()
+        logging.info("Updated path to Kedro project: %s", str(project_path))
+    else:
+        logging.info("No path argument was provided. Using: %s", str(project_path))
+
     path = path or project_path
```

Comment:
I don't see the need for this line any more - should we remove it and just use `project_path` in all following code?